common/lib/xmodule/xmodule/js/fixtures/video_all.html

```diff
@@ -32,7 +33,14 @@
             <section class="video-player">
               <div id="id"></div>
             </section>
+            <a href="#" tabindex="0" class="hide-captions" role="button" aria-disabled="false"
+                  title="${_('Turn off transcripts')}">
+              <i class="icon-caret-right" aria-hidden="true"></i>
```

Comment:
We've recently updated our Font Awesome library, so you'll want to update the icons to use the [latest Font Awesome set] (http://fontawesome.io/cheatsheet/). A rebase against Master might be wise too.